apps/frontend/web/src/pages/GameSettings.tsx

```diff
@@ -0,0 +1,309 @@
+    try {
+      await fetch(`/api/games/${pendingGameData.id}`, {
+        method: 'DELETE',
+      });
+      toast({
+        title: 'Game deleted.',
+        description: "We're sorry to see you go!",
+        status: 'error',
+        duration: 2000,
+        isClosable: true,
+      });
+    } catch (error) {
+      toast({
+        title: 'There was an error deleting your game.',
+        status: 'error',
+        duration: 2000,
+        isClosable: true,
+      });
+    }
+  };
+
+  const updateGameSettings = async () => {
+    try {
+      const req = await fetch(`/api/games/${pendingGameData.id}`, {
+        method: 'PATCH',
+        body: JSON.stringify(pendingGameData),
+        headers: { 'Content-type': 'application/json' },
+      });
+      const res = await req.json();
+
+      if (req.ok) {
+        setGameData(res);
+        toast({
+          title: 'Informations updated.',
+          status: 'success',
+          duration: 2000,
+          isClosable: true,
+        });
+      }
+    } catch {}
+  };
+
+  return (
+    <>
+      <Navbar />
+      <Flex
+        flexDir="column"
+        bgColor={{
+          base: `${pageColor}.200`,
+          xl: 'white',
+          lg: `${pageColor}.200`,
+          md: `${pageColor}.200`,
+          sm: `${pageColor}.200`,
+        }}
+        minH="100vh"
+        alignItems="center"
+      >
+        <Flex
+          flexDir="column"
+          alignItems="center"
+          py="3rem"
+          bgColor="#FFF"
+          w="100%"
+        >
+          <Heading color={`${pageColor}.900`}>{'Game Settings'}</Heading>
+        </Flex>
+        <Flex w="100%">
+          <Flex
+            justifyContent="space-around"
+            bgColor={{
+              base: `${pageColor}.200`,
+              xl: 'white',
+              lg: `${pageColor}.200`,
+              md: `${pageColor}.200`,
+              sm: `${pageColor}.200`,
+            }}
+            flexGrow="1"
+          >
+            <VStack
+              bgColor={`${pageColor}.200`}
+              boxSize={{ base: 'md', xl: 'xl' }}
+              w="auto"
+              h="auto"
+              p="10"
+            >
+              <Text display={displayDesktop} as="b" fontSize="xl" mb="5">
+                {`Change your informations`}
+              </Text>
+
+              <Image
+                display={displayMobile}
+                w="5.5rem"
+                src={`/${gameData?.image?.name}.png`}
+                alt={`Image of ${gameData?.image?.description}`}
+                mt="2rem"
+                mb="1rem"
+              />
+              <Button
+                display={displayMobile}
+                onClick={gameImage.onOpen}
+                bgColor={`${pageColor}.900`}
+                color="#FFF"
+                mb={marginTopButton}
+                w="8rem"
+                fontWeight="normal"
+                boxShadow="rgb(0 0 0 / 40%) 0px 3px 5px"
+              >
+                {'Modify'}
```

Comment:
```suggestion
                {'Edit'}
```
Fais une recherche sur tout le projet pour remplacer "Modify" par "Edit" au cas où j'en ai oublié

Reply:
Fait